DemoScript.sh

```diff
@@ -0,0 +1,2 @@
+export PROJECT_TOKEN=<INSERT_PROJECT_TOKEN>
+mvn -P ExecuteTestSuite test
```

Comment:
Looks like the command `mvn -P ExecuteTestSuite test` doesn't work, At least for me like it's not launching tests. Is it working for you?

Reply:
Yes it is working in my case.